lib/web/service-worker/service-worker-global-scope.js

```diff
@@ -0,0 +1,84 @@
+import { parentPort } from 'node:worker_threads'
+import { Clients, kAddClient } from './clients.js'
+import { ServiceWorker } from './service-worker.js'
+import { Client } from './client.js'
+import { CacheStorage } from '../cache/cachestorage.js'
```

Comment:
TIL Undici has its own `CacheStorage` implementation. I'm reusing it for the `ServiceWorkerGlobalScope` as-is.